Hogwarts/src/main/java/com/example/Hogwarts/controller/FacultyController.java

```diff
@@ -0,0 +1,66 @@
+package com.example.Hogwarts.controller;
+
+import com.example.Hogwarts.model.Faculty;
+import com.example.Hogwarts.model.Student;
+import com.example.Hogwarts.service.FacultyService;
+import com.example.Hogwarts.service.StudentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/faculty")
+public class FacultyController {
+
+    private final FacultyService facultyService;
+
+    private Faculty faculty;
+
+    @Autowired
+    public FacultyController(FacultyService facultyService, StudentService studentService) {
```

Comment:
StudentService в этом контроллере не нужен, у тебя он просто добавлен в конструктор и нигде не используется
```suggestion
    public FacultyController(FacultyService facultyService) {
```